### PR TITLE
Issue 429 by Pjohans:update Ding node search feature to show an empty vi...

### PR DESCRIPTION
**Pull request by pjohans**
...ew when no results are found.

I have set the Ding node search views exposed form to 'input required' to force some output. This pull-request works in conjunction with the next pull request on ddbasic

### CODE REVIEW

File: ding_frontend.views_default.inc [hunk 1]
```diff
@@ -29,12 +29,14 @@ function ding_frontend_views_default_views() {
...
   $handler->display->display_options['query']['type'] = 'views_query';
   $handler->display->display_options['query']['options']['query_comment'] = FALSE;
-  $handler->display->display_options['exposed_form']['type'] = 'basic';
+  $handler->display->display_options['exposed_form']['type'] = 'input_required';
   $handler->display->display_options['exposed_form']['options']['submit_button'] = 'Udfør';
   $handler->display->display_options['exposed_form']['options']['reset_button_label'] = 'Gendan';
   $handler->display->display_options['exposed_form']['options']['exposed_sorts_label'] = 'Sortér efter';
   $handler->display->display_options['exposed_form']['options']['sort_asc_label'] = 'Stigende';
   $handler->display->display_options['exposed_form']['options']['sort_desc_label'] = 'Faldende';
+  $handler->display->display_options['exposed_form']['options']['text_input_required'] = 'Der blev ikke fundet nogle resultater';
```
Comment (cableman) [inline ding_frontend.views_default.inc]: Please ensure that all strings are in English so they can be translated correctly.

File: ding_frontend.views_default.inc [hunk 2]
```diff
@@ -105,6 +107,7 @@ function ding_frontend_views_default_views() {
     t('Sortér efter'),
     t('Stigende'),
     t('Faldende'),
+    t('Der blev ikke fundet nogle resultater'),
```
Comment (cableman) [inline ding_frontend.views_default.inc]: The string from the comment above should be in english.

Comment (pjohans) [inline ding_frontend.views_default.inc]: right

Comment (guddo) [inline ding_frontend.views_default.inc]: Change your backend users language to English and you should be safe.
If that doesn't work, try disabling the Danish language.

File: ding_frontend.views_default.inc [hunk 3]
```diff
@@ -53,19 +55,19 @@ function ding_frontend_views_default_views() {
   $handler->display->display_options['row_plugin'] = 'node';
   $handler->display->display_options['row_options']['view_mode'] = 'search_result';
   $handler->display->display_options['row_options']['links'] = FALSE;
-  /* Field: Content: Title */
+  /* Felt: Indhold: Titel */
```
Comment (cableman) [inline ding_frontend.views_default.inc]: Language have changed to Danish?

File: ding_frontend.views_default.inc [hunk 4]
```diff
@@ -53,19 +55,19 @@ function ding_frontend_views_default_views() {
...
   $handler->display->display_options['fields']['title']['id'] = 'title';
   $handler->display->display_options['fields']['title']['table'] = 'node';
   $handler->display->display_options['fields']['title']['field'] = 'title';
   $handler->display->display_options['fields']['title']['label'] = '';
   $handler->display->display_options['fields']['title']['alter']['word_boundary'] = FALSE;
   $handler->display->display_options['fields']['title']['alter']['ellipsis'] = FALSE;
-  /* Sort criterion: Content: Updated date */
+  /* Sorteringskriterie: Indhold: Opdateringsdato */
```
Comment (cableman) [inline ding_frontend.views_default.inc]: Language have changed to Danish?

File: ding_frontend.views_default.inc [hunk 5]
```diff
@@ -53,19 +55,19 @@ function ding_frontend_views_default_views() {
...
   $handler->display->display_options['sorts']['changed']['id'] = 'changed';
   $handler->display->display_options['sorts']['changed']['table'] = 'node';
   $handler->display->display_options['sorts']['changed']['field'] = 'changed';
   $handler->display->display_options['sorts']['changed']['order'] = 'DESC';
-  /* Contextual filter: Search: Search Terms */
+  /* Kontekstuelt filter: Søg: Søgeord */
```
Comment (cableman) [inline ding_frontend.views_default.inc]: Language have changed to Danish?

File: ding_frontend.views_default.inc [hunk 6]
```diff
@@ -75,14 +77,14 @@ function ding_frontend_views_default_views() {
   $handler->display->display_options['arguments']['keys']['summary']['number_of_records'] = '0';
   $handler->display->display_options['arguments']['keys']['summary']['format'] = 'default_summary';
   $handler->display->display_options['arguments']['keys']['summary_options']['items_per_page'] = '25';
-  /* Filter criterion: Content: Published */
+  /* Filterkriterie: Indhold: Udgivet */
```
Comment (cableman) [inline ding_frontend.views_default.inc]: Language have changed to Danish?

File: ding_frontend.views_default.inc [hunk 7]
```diff
@@ -75,14 +77,14 @@ function ding_frontend_views_default_views() {
...
   $handler->display->display_options['filters']['status']['id'] = 'status';
   $handler->display->display_options['filters']['status']['table'] = 'node';
   $handler->display->display_options['filters']['status']['field'] = 'status';
   $handler->display->display_options['filters']['status']['value'] = 1;
   $handler->display->display_options['filters']['status']['group'] = 0;
   $handler->display->display_options['filters']['status']['expose']['operator'] = FALSE;
-  /* Filter criterion: Content: Type */
+  /* Filterkriterie: Indhold: Type */
```
Comment (cableman) [inline ding_frontend.views_default.inc]: Language have changed to Danish?

**Comment by pjohans** [comment]
now in english

**Pull request merged**